src/test/java/LoginTests.java

```diff
@@ -1,55 +1,62 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
+import pages.LoginPage;
 
 
 public class LoginTests extends BaseTest {
 
-    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class, enabled = true, priority = 0, description = "Login with invalid email and valid password")
-    public void loginInvalidEmailValidPassword(String email, String password) throws InterruptedException {
-
-        provideEmail(email);
-        providePassword(password);
-        clickSubmit();
-
-        Thread.sleep(2000);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
-
-    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class)
-    public void loginEmptyEmailPassword(String email, String password) throws InterruptedException {
-        provideEmail(email);
-        providePassword(password);
-        clickSubmit();
-
-        Thread.sleep(2000);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
-    
+//    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class, enabled = true, priority = 0, description = "Login with invalid email and valid password")
+//    public void loginInvalidEmailValidPassword(String email, String password) throws InterruptedException {
+//
+//        provideEmail(email);
+//        providePassword(password);
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
+//
+//    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class)
+//    public void loginEmptyEmailPassword(String email, String password) throws InterruptedException {
+//        provideEmail(email);
+//        providePassword(password);
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
+//
+//    @Test (enabled = true, priority = 1, description = "Login with valid email and valid password")
+//    public void loginValidEmailPassword(){
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//        isAvatarDisplayed();
+//    }
+//
+//    @Test (enabled = true, priority = 3, description = "Login with valid email and empty password")
+//    public void loginValidEmailEmptyPassword() throws InterruptedException {
+//
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("");
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url); //https://qa.koel.app/
+//    }
+//    public void isAvatarDisplayed() {
+//        WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+//        Assert.assertTrue(avatarIcon.isDisplayed());
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.